client/core/core.go

```diff
@@ -50,6 +51,8 @@ const (
 	// tickCheckDivisions is how many times to tick trades per broadcast timeout
 	// interval. e.g. 12 min btimeout / 8 divisions = 90 sec between checks.
 	tickCheckDivisions = 8
+	// Version is the core version.
+	Version = 1
 )
```

Comment:
The upgrade required us to have the old `Crypter` in-hand, so couldn't be done through a database upgrade. This PR introduces core versions and "authenticated upgrades" that are performed in `Login`.